resources/views/auth/login.blade.php

```diff
@@ -32,17 +32,35 @@ function showGitHubAuthWindow() {
                 <h2 class="title mb-8">
                     Log in
                 </h2>
-                <h3 class="title-sm mb-4">
-                    Via GitHub
-                </h3>
-                <a href="{{ route('github-login') }}" onclick="event.preventDefault(); showGitHubAuthWindow()">
-                    <x-button>
-                        <span class="mr-3 icon w-6 opacity">
+
+                <div class="grid grid-cols-1 sm:grid-cols-2 gap-12">
+                    <div>
+                        <h3 class="title-sm mb-4">
```

Comment:
@riasvdv Maybe we can drop these headings?

Reply:
Yeah that's better!